lib/services/multipart-upload.ts

```diff
@@ -11,6 +11,7 @@ export class MultipartUploadService implements IMultipartUploadService {
 	private static INPUT_FILE_ENCODING = "binary";
 	private static HASH_ALGORITHM = "sha512";
 	private static HASH_ENCODING = "base64";
+	private static MAX_CONCURRENT_FUTURES = 3;
```

Comment:
We've discussed it with @ivanovit  and it turned out there were some issues when multiple connections are initialized, so we were thinking for 3 or 5 concurrent uploads. But at the end we've decided to use 3, just to assure there will be no more (or only a few) errors caused by simultaneous connections.